backend/test/fixtures/enrollments.yml

```diff
@@ -1,73 +1,40 @@
 DEFAULTS: &DEFAULTS
-  target_api: "franceconnect"
-  previous_enrollment_id: null
-  copied_from_enrollment_id: null
+  siret: "83906810300010"
+  nom_raison_sociale: "BIG COMPANY©"
```

Comment:
A fresh install should be filled with data that is likely to be found in production. Plz update these data accordingly.